ddtrace/appsec/processor.py

```diff
@@ -27,11 +29,47 @@ def get_rules():
     return get_env("appsec", "rules", default=DEFAULT_RULES)
 
 
+COLLECTED_REQUEST_HEADERS = {
+    "accept",
+    "accept-encoding",
+    "accept-language",
+    "content-encoding",
+    "content-language",
+    "content-length",
+    "content-type",
+    "forwarded",
+    "forwarded-for",
+    "host",
+    "true-client-ip",
+    "user-agent",
+    "via",
+    "x-client-ip",
+    "x-cluster-client-ip",
+    "x-forwarded",
+    "x-forwarded-for",
+    "x-real-ip",
+}
+
+COLLECTED_HEADER_PREFIX = "http.request.headers."
+
+
+def _set_headers(span, headers):
+    for k in headers:
+        if k in COLLECTED_REQUEST_HEADERS:
+            span._set_str_tag(COLLECTED_HEADER_PREFIX + k.lower(), headers[k])
```

Comment:
you check that `k` is in `COLLECTED_REQUEST_HEADERS` but you don't `lower` it while you do it on L59.
That confuses me.
You want to `lower` it all the time, no?